parm/config/config.base.emc.dyn

```diff
@@ -284,5 +284,18 @@ export ARCH_WARMICFREQ=4     # Archive frequency in days for warm_start capabili
 export ARCH_FCSTICFREQ=1     # Archive frequency in days for gdas and gfs forecast-only capability
 
 export DELETE_COM_IN_ARCHIVE_JOB="YES"   # NO=retain ROTDIR.  YES default in arch.sh and earc.sh.
+if [ $machine = "HERA" ]; then
+    export FIX_AER="/scratch2/NCEPDEV/climate/Anning.Cheng/MERRA2"
+    export FIX_LUT="/scratch2/NCEPDEV/climate/Anning.Cheng/aer_data/LUTS"
+elif [ $machine = "WCOSS_C" ]; then
+    export FIX_AER="/gpfs/dell2/emc/modeling/noscrub/Anning.Cheng/MERRA2"
+    export FIX_LUT="/gpfs/dell2/emc/modeling/noscrub/Anning.Cheng/aer_data/LUTS"
+elif [ $machine = "WCOSS_DELL_P3" ]; then
+    export FIX_AER="/gpfs/dell2/emc/modeling/noscrub/Anning.Cheng/MERRA2"
+    export FIX_LUT="/gpfs/dell2/emc/modeling/noscrub/Anning.Cheng/aer_data/LUTS"
+elif [ $machine = "ORION" ]; then
+    export FIX_AER="/work/noaa/global/acheng/MERRA2"
+    export FIX_LUT="/work/noaa/global/acheng/aer_data/LUTS"
+fi
```

Comment:
Looks like new fix files to support Merra2? If so I will pull them into the main $FIX_DIR sets on supported platforms for you. Then this block can be removed and replaced by a single FIX_AER and FIX_LUT path setting to the subfolder in $FIX_DIR.

Take a look at one of the following $FIX_DIRs and let me know where these new fix files should go within:
Hera: /scratch1/NCEPDEV/global/glopara/fix/
WCOSS-Dell: /gpfs/hps3/emc/global/noscrub/emc.glopara/git/fv3gfs/fix/
Orion: /work/noaa/global/glopara/fix

Thanks!